tests/drivers/spi/spi_loopback/overlay-stm32-spi-16bits.conf

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2023 Graphcore Ltd, All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_SPI_LOOPBACK_16BITS_FRAMES=y
```

Comment:
Do you expect some specific value for CONFIG_SPI_STM32_INTERRUPT ?
In this case, I think it should be specified.

FYI, there is a bit of inconsistency on STM32 SPI interrupt config setting.
It is enabled in lot of board configurations, but not all. We need to fix this at one point.

Reply:
> Do you expect some specific value for CONFIG_SPI_STM32_INTERRUPT ?

This configuration file intends to test non-DMA, non-interrupts SPI operations, so I specifically want it to be disabled.

> In this case, I think it should be specified.

It's now `CONFIG_SPI_STM32_INTERRUPT=n`, I imagine that's what you were referring to?